SECURITY.md

```diff
@@ -16,29 +16,93 @@ You may need to [update the security provider](https://developer.android.com/tra
 
 ## TLS with OpenSSL
 
-This is currently the recommended approach for using gRPC over TLS (on non-Android systems). 
+This is currently the recommended approach for using gRPC over TLS (on non-Android systems).
 
-### Benefits of using OpenSSL
+The main benefits of using OpenSSL are:
 
 1. **Speed**: In local testing, we've seen performance improvements of 3x over the JDK. GCM, which is used by the only cipher suite required by the HTTP/2 spec, is 10-500x faster.
 2. **Ciphers**: OpenSSL has its own ciphers and is not dependent on the limitations of the JDK. This allows supporting GCM on Java 7.
 3. **ALPN to NPN Fallback**: if the remote endpoint doesn't support ALPN.
 4. **Version Independence**: does not require using a different library version depending on the JDK update.
 
-### Requirements for using OpenSSL
+Support for OpenSSL is only provided for the Netty transport via [netty-tcnative](https://github.com/netty/netty-tcnative), which is a fork of
+[Apache Tomcat's tcnative](http://tomcat.apache.org/native-doc/), a JNI wrapper around OpenSSL. As of version `1.1.33.Fork13`, netty-tcnative provides two options for usage: statically or dynamically linked. For simplification of setup,
+we recommend using `netty-tcnative-boringssl-static`, which is statically linked against BoringSSL and Apache APR. Using this artifact requires no extra installation and guarantees that ALPN and the ciphers required for
```

Comment:
Netty-tcnative-boringssl-static sounds convenient, but perhaps netty-tcnative might be more suitable for production environments? In the event of an important security update to OpenSSL/BoringSSL, with netty-tcnative we would basically just run "apt-get upgrade" and be done. With netty-tcnative-boringssl-static we would have to:

* Wait for Netty team to release a new version of the netty-tcnative-boringssl-static that includes the fix.
* Possibly update projects to depend on the new netty-tcnative-boringssl-static version.
* Possibly wait for it to appear on Maven Central.
* Rebuild all projects that use GRPC to pick up the new netty-tcnative-boringssl-static JAR.
* Push the new release(s).

We would also have to find out that an important new fix is available in the first place, which would be nontrivial and error-prone because package managers such as apt-get won't say anything about new releases of netty-tcnative-boringssl-static.

With netty-tcnative, patching is almost immediate and effortless. With netty-tcnative-boringssl-static, there is a longer time before servers are patched, and there is more scope for things to go wrong such that servers *never* get patched. Saying "we recommend using netty-tcnative-boringssl-static" is thus recommending the more risky option.

Meanwhile, there's little difference in setup difficulty, because netty-tcnative just needs an extra "apt-get install libapr1" step. It's a small price to pay.

Is there some other benefit to using netty-tcnative-boringssl-static?